### PR TITLE
[Cookbook] Add note about possible 404 error on Heroku

### PR DESCRIPTION
**Pull request by bicpi**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | no |
| Applies to | all |
| Fixed tickets | - |

I was running into this 404 and thought something went wrong with the doc root setting first.

### CODE REVIEW

File: cookbook/deployment/heroku.rst [hunk 1]
```diff
@@ -208,6 +208,14 @@ application will respond:
 
 You should be seeing your Symfony application in your browser.
 
+.. caution::
+
+    If you take your first steps on Heroku using a fresh installation of
+    the Symfony Standard Edition you may run into a 404 page not found error.
+    This is because the route for `/` is defined by the AcmeDemoBundle, but the
```
Comment (javiereguiluz) [inline cookbook/deployment/heroku.rst]: minor syntax issue:

```
 `/`  should be ``/``
```

Comment (javiereguiluz) [inline cookbook/deployment/heroku.rst]: minor syntax issue:

```
 `/app/example`  should be ``/app/example``
```

**Comment by javiereguiluz** [comment]
:+1: @bicpi thanks for this improvement.

**Comment by bicpi** [comment]
@javiereguiluz Fixed the backticks, thanks for the hint.

File: cookbook/deployment/heroku.rst [hunk 2]
```diff
@@ -208,6 +208,14 @@ application will respond:
 
 You should be seeing your Symfony application in your browser.
 
+.. caution::
+
+    If you take your first steps on Heroku using a fresh installation of
+    the Symfony Standard Edition you may run into a 404 page not found error.
```
Comment (xabbuh) [inline cookbook/deployment/heroku.rst]: [...] Standard Edition, you may [...]

**Comment by WouterJ** [comment]
I like this one. However, this is all going to change (acmedemobundle will be removed in favor of a symfony demo application and the appbundle will default to /). But I think we should merge this PR soon, to avoid further confusion, and see what's going to happen in the future.

**Comment by javiereguiluz** [comment]
@WouterJ I totally agree with you: merge it ASAP to avoid confusion and let's see what happens in the future.

**Comment by xabbuh** [comment]
:+1:

**Comment by bicpi** [comment]
Is the AcmeDemoBundle going to be removed from the 2.3 branch, too
Sorry, I missed the discussion about this step.

**Comment by xabbuh** [comment]
@bicpi Yes (see sensiolabs/SensioDistributionBundle@96d645f311eb062b52c26ffbd90bedd007e0f565)

**Comment by bicpi** [comment]
@xabbuh Thanks!

**Comment by weaverryan** [comment]
Cool - I agree with everything you guys said :). Thanks!

**Pull request merged**